### PR TITLE
Backside cable management for V2.4 350mm

### PR DESCRIPTION
**Pull request by jbisson9**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Printer Mod to manage wiring going up backside of printer to exhaust fan and BTT smart filament sensor

#### How was it tested? 

Tested on my own machine.

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ like so:
 | Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |:x:|
 | jatson | [Plug panel mod for Schurter CD44.1101.151 power entry](./jatson/plug_panel_schurter) | A modified plug panel for Voron 2.4r1 with Schurter CD44.1101.151 power entry modul. The panel has a LAN socket as well. | :x: | :grey_question: | :heavy_check_mark: | :x: |:x:|
 | jaysuk | [4.3 Fly Screen Mount](./jaysuk/4.3_fly_screen_mount) | A mount for a 4.3" Fly Screen | :heavy_check_mark: | :x: | :x: | :x: |:x:|
+| jbisson9 | [Backside Cable Management](./jbisson9/Backside_Cable_Management) | Routing wiring for the exhaust filter and BBT smart filament sensor | :x: | :grey_question: | :heavy_check_mark: | :x: | :grey_question:  |
```
Comment (MapleLeafMakers) [inline printer_mods/README.md:208]: typo here, `BBT` should be `BTT`

File: printer_mods/jbisson9/Backside_Cable_Management/CAD/.gitkeep [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (MapleLeafMakers) [inline printer_mods/jbisson9/Backside_Cable_Management/CAD/.gitkeep:1]: This file should be removed.

File: printer_mods/jbisson9/Backside_Cable_Management/Images/.gitkeep [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (MapleLeafMakers) [inline printer_mods/jbisson9/Backside_Cable_Management/Images/.gitkeep:1]: This file should be removed.

File: printer_mods/jbisson9/Backside_Cable_Management/STLs/.gitkeep [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (MapleLeafMakers) [inline printer_mods/jbisson9/Backside_Cable_Management/STLs/.gitkeep:1]: This file should be removed.

File: printer_mods/jbisson9/Backside_Cable_Management/Readme.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* 2-, 3- and 5-circuit microfit 3 connectors
+* Optional M3 heat set insert and M3 x 12 SCHS
+
+# Printing
+
+Print what you need.  Uses standard Voron print profiles.  Supports not needed but in reality the BTT jig could use it (sorry first design and I don't have all my sh*^Y#t together)
+
+# Assembly
+
+Attach the BTT sensor to the jig with two M3 button head screws.  Create the wiring harnesses to attach the microfit connectors.  As an aside, all three ends are males connectors, unusual generally, but easier to build jigs around them.
```
Comment (MapleLeafMakers) [inline printer_mods/jbisson9/Backside_Cable_Management/Readme.md:35]: These instructions are a bit unclear.  There are no stls with 'jig' anywhere in the filename. I'm not sure what part 'the jig' refers to -- `microfit_jig.jpg` and `completed_jig.jpg` both show parts that don't have any obvious way to connect to the btt sensor, is this a different jig?

**Review by MapleLeafMakers — COMMENTED** [review]
STLs need to be oriented correctly for printing, most of the `backend_*` files are not, for example: 
image

File: printer_mods/jbisson9/Backside_Cable_Management/Readme.md (hunk 5 above)
Comment (jbisson9) [reply printer_mods/jbisson9/Backside_Cable_Management/Readme.md]: @MapleLeafMakers going to rename these images and assembly to clam shells.

File: printer_mods/README.md (hunk 1 above)
Comment (jbisson9) [reply printer_mods/README.md]: @MapleLeafMakers done

File: printer_mods/jbisson9/Backside_Cable_Management/CAD/.gitkeep (hunk 2 above)
Comment (jbisson9) [reply printer_mods/jbisson9/Backside_Cable_Management/CAD/.gitkeep]: @MapleLeafMakers done

File: printer_mods/README.md (hunk 1 above)
Comment (jbisson9) [reply printer_mods/README.md]: @MapleLeafMakers fixed the stls and readme

**Comment by jbisson9** [comment]
@MapleLeafMakers, I added Canbus cabling into the system and modified certain pieces.  How do I recall this pull request or do I simply add to the existing PR?

**Comment by jbisson9** [comment]
@MapleLeafMakers looking to complete this PR if you're good with it

**Pull request closed without merging**